test/comparator/test_file_set_comparator.py

```diff
@@ -157,10 +157,11 @@ def test_message_in_dependency_change_breaking(self):
         ).compare()
         # The breaking change should be in field level, instead of message removal,
         # since the message is imported from dependency file.
-        finding = self.finding_container.get_all_findings()[0]
-        self.assertEqual(finding.change_type.name, "MAJOR")
-        self.assertEqual(finding.category.name, "FIELD_TYPE_CHANGE")
-        self.assertEqual(finding.location.proto_file_name, "update.proto")
+        breaking = self.finding_container.get_actionable_findings()
```

Comment:
nit: does the comment above need to be updated?

Reply:
Done (here and in another similar place in the same file)